app_integrations/apps/app_base.py

```diff
@@ -259,11 +233,11 @@ def _finalize(self):
                          'due to the subclass not setting this value.')
 
         if self._last_timestamp == self._config.start_last_timestamp:
-            LOGGER.error('Ending last timestamp is the same as the beginning last timestamp. '
-                         'This could occur if there were no logs collected for this execution.')
+            LOGGER.warning('Ending last timestamp is the same as the beginning last timestamp. '
```

Comment:
I don't think this is considered to be an abnormal behavior. I would use `info` logging level.

Reply:
thanks - I will change it!